crates/jstzd/src/network.rs

```diff
@@ -0,0 +1,60 @@
+use async_dropper::{AsyncDrop, AsyncDropper};
+use async_trait::async_trait;
+use bollard::{network::CreateNetworkOptions, Docker};
+use std::sync::Arc;
+
+#[derive(Default)]
+struct NetworkInner {
+    name: String,
+    client: Option<Arc<Docker>>,
```

Comment:
Ditto, why is this optional? Suggest to make it non-optional

Reply:
Also remove boilerplate using derive for AsyncDrop

Reply:
> Ditto, why is this optional? Suggest to make it non-optional

I had to make this `optional` as the AsyncDropper required the `Default` trait. However now it's disabled so can remove it thanks for the suggestion! @zcabter

Reply:
> Also remove boilerplate using derive for AsyncDrop

is there any example for this? doesn't seem like i can derive using `async_dropper_simple`